pkgs/applications/misc/plasma-theme-switcher/default.nix

```diff
@@ -0,0 +1,41 @@
+{
+  stdenv, lib, pkgs, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "plasma-theme-switcher";
+  version = "v0.1";
+  dontWrapQtApps = true;
+
+  src = fetchFromGitHub {
+    owner = "maldoinc";
+    repo = "plasma-theme-switcher";
+    rev = version;
+    sha256 = "sdcJ6K5QmglJEDIEl4sd8x7DuCPCqMHRxdYbcToM46Q=";
+  };
+
+  buildInputs = [
+    pkgs.cmake
+    pkgs.extra-cmake-modules
+    pkgs.qt5.full
+    pkgs.libsForQt5.plasma-framework
+  ];
+
+  buildPhase = ''
```

Comment:
can be omitted, just add `cmake` and `extra-cmake-modules` to `nativeBuildInputs` (and remove from `buildInputs`)